frontend/components/Settings/ClubTab.js

```diff
@@ -66,55 +62,54 @@ export default props => {
         </tr>
       </thead>
       <tbody>
-        {isMemberOfAnyClubs ? (
-          <tr>
-            <td className="has-text-grey" colSpan="4">
-              You are not a member of any clubs yet.
+        {userInfo.membership_set.map(item => (
+          <tr key={item.code}>
+            <td>
+              <Link
+                href="/club/[club]"
+                as={`/club/${item.code}`}
+              >
+                <a>{item.name}</a>
+              </Link>
             </td>
-          </tr>
-        ) : (
-          userInfo.membership_set.map(item => (
-            <tr key={item.code}>
-              <td>
-                <Link
-                  href="/club/[club]"
-                  as={`/club/${item.code}`}
+            <td>{item.title}</td>
+            <td>{item.role_display}</td>
+            <td>
+              <Toggle
+                club={item}
+                active={item.active}
+                toggle={club => toggleActive(club)}
+              />
+            </td>
+            <td>
+              <Toggle
+                club={item}
+                active={item.public}
+                toggle={club => togglePublic(club)}
+              />
+            </td>
+            <td>
+              {item.role_display === 'Admin' ? (
+                <button className="button is-small">Manage</button>
+              ) : (
+                <button
+                  className="button is-small"
+                  onClick={() => leaveClub(item)}
                 >
-                  <a>{item.name}</a>
-                </Link>
-              </td>
-              <td>{item.title}</td>
-              <td>{item.role_display}</td>
-              <td>
-                <Toggle
-                  club={item}
-                  active={item.active}
-                  toggle={club => toggleActive(club)}
-                />
-              </td>
-              <td>
-                <Toggle
-                  club={item}
-                  active={item.public}
-                  toggle={club => togglePublic(club)}
-                />
-              </td>
-              <td>
-                {item.role_display === 'Admin' ? (
-                  <button className="button is-small">Manage</button>
-                ) : (
-                  <button
-                    className="button is-small"
-                    onClick={() => leaveClub(item)}
-                  >
                     Leave
-                  </button>
-                )}
-              </td>
-            </tr>
-          ))
-        )}
+                </button>
+              )}
+            </td>
+          </tr>
+        ))}
       </tbody>
     </Table>
+  ) : (
+      <>
+        <EmptyState name="button" />
+        <p className="has-text-grey" style={{ textAlign: 'center' }}>
+          You are not a member of any clubs yet.
+        </p>
```

Comment:
Maybe refactor this to use the `<Text .. />` component? Could add a `isGray` and `isCentered` props to it if it doesn't already have that functionality baked in.